pyproject.toml

```diff
@@ -31,11 +31,19 @@ pyflakes = "^3.0.1"
 pytest = ">=7.1.3,<9.0.0"
 pytest-asyncio = ">=0.21,<0.24"
 protoc-gen-validate = ">=0.4.1,<=1.0.4"
+types-protobuf = "^4.25"
 
 [tool.black]
 exclude = '(^\.(.*)|^(.*)_pb2(_grpc)?\.py)'
 line-length = 100
 
+[tool.mypy]
+explicit_package_bases = true
+
+[[tool.mypy.overrides]]
+ignore_missing_imports = true
+module = ["google.rpc.*", "grpcutil"]
```

Comment:
Types for these modules don't seem to exist ... anywhere so I just told mypy to ignore them